### ISSUE TITLE
[Accessibility/Usability] Handle onEnter in core.prompt dialogs

### ISSUE THREAD

**Issue by microsoftsam** [Microsoft/pxt#2459]
Dialogs used for Blockly.Prompt (eg: Create a variable) don't handle the onEnter key pressed. This means the user has to click the OK button everytime after entering their input.

**Comment by veler** [comment]
Can't reproduce this problem. Typing Enter on the OK button make the modal close. But I realized that the tabbing trap doesn't work for this modal. It's another issue

**Comment by microsoftsam** [comment]
@veler we want to handle enter on the input field. eg: a user can type the var name and hit enter, rather than having to tab first to the ok button and hit enter. (enter should do the approve action of a modal prompt dialog)

**Comment by veler** [comment]
Oh OK ! I though you were talking about the Enter key on the OK button. I will work on it later.

**Comment by veler** [comment]
Bug fixed and included to [https://github.com/Microsoft/pxt/pull/2440](https://github.com/Microsoft/pxt/pull/2440)

**Comment by veler** [comment]
Reopened to push the fix directly into Master.
